charts/opentelemetry-operator/values.schema.json

```diff
@@ -1571,11 +1571,20 @@
                             "examples": [
                                 true
                             ]
+                        },
+                        "certPeriodYear": {
```

Comment:
why are we using years here if we're just going to convert this to days? IMO we should only use days

Reply:
IMO we should stick with days as its more granular, ex: i have expirations set to 290 days for some of my company.

Reply:
oh, day more flexible, I will update it.